spec/lib/formatters/text_lines_formatter_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "spec_helper"
+
+describe SimpleCovLinterFormatter::TextLinesFormatter do
+  let(:command_name) { "RSpec" }
+  let(:lines) do
+    [
+      "file1.rb:1:1:missed-2",
+      "file1.rb:2:1:never-2",
+      "file2.rb:1:1:missed-4",
+      "file2.rb:2:1:covered-4",
+      "file2.rb:4:1:missed-4",
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Style/TrailingCommaInArrayLiteral: Avoid comma after the last item of an array.